src/admin/vote/Edit.jsx

```
@@ -20,15 +192,212 @@
+        placeholder="In der Schülerprojektwoche vom 12. bis 16. Juli 2024 werden von SchülerInnen organisierte Projekte angeboten. Die Projekte finden täglich von 10:00 bis 15:00 in den angegebenen Räumen statt."
+        rows={3}
+        maxlength={200}
+        counter
+        value={description}
+        onInput={(e) => setDescription(e.target.value)}
+      ></mdui-text-field>
+      <p />
+      {extraFields.map((e, i) => (
+        <>
+          <div className="fields-row">
+            <mdui-text-field
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)